sys-fs/mdadm/files/mdadm.service

```
@@ -0,0 +1,5 @@
+[Unit]
+Description=Initiates a check run of an MD array's redundancy information.
+
+[Service]
+ExecStart=/usr/sbin/checkarray
```

```suggestion
ExecStart=/usr/sbin/checkarray --cron --all --idle --quiet
```
I think we should keep the same flags